### PR TITLE
Implement create trials screen

### PR DESCRIPTION
**Pull request by Peshp**

### CODE REVIEW

**Review by HristoPetkov94 — CHANGES_REQUESTED** [review]

File: lib/screens/create_trials/create_trials_screen.dart [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import 'dart:convert';
+import 'dart:html';
```
[the remark this answers is not in the document]
Comment (Peshp) [reply lib/screens/create_trials/create_trials_screen.dart]: I've removed that import.

File: lib/screens/create_trials/create_trials_screen.dart [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        backgroundColor: Colors.lightBlue.shade600,
+      ),
+      body: Expanded(
+      child: Column(
+        children: [
+          Container(
+            padding: const EdgeInsets.all(16.0),
+            child: const Text('Doctor name'),
+          ),
+          Expanded(
```
[the remark this answers is not in the document]
Comment (Peshp) [reply lib/screens/create_trials/create_trials_screen.dart]: I've removed every expanded widget.

File: pubspec.yaml [hunk 3]
```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.6
+  syncfusion_flutter_pdf: ^25.2.6
```
[the remark this answers is not in the document]
Comment (Peshp) [reply pubspec.yaml:38]: I've removed escort to pdf button, because is not needed for this task.

Comment (HristoPetkov94) [inline pubspec.yaml:38]: remove the dependency aswell

**Review by HristoPetkov94 — APPROVED** [review]

**Pull request merged**